src/pixelator/analysis/__init__.py

```diff
@@ -45,15 +44,16 @@ def analyse_pixels(
     compute_polarization: bool,
     compute_colocalization: bool,
     use_full_bipartite: bool,
-    polarization_normalization: Literal["raw", "clr", "denoise"],
-    polarization_binarization: bool,
+    polarization_normalization: Literal["raw", "clr"],
```

Comment:
Is this `PolarizationNormalizationTypes`?

Reply:
Yes! Fixed!